src/common/options.cc

```diff
@@ -2570,7 +2570,7 @@ std::vector<Option> get_global_options() {
     .add_service("mon"),
 
     Option("osd_pool_default_pg_num", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
-    .set_default(16)
+    .set_default(32)
```

Comment:
might want to update dashboard tests accordingly
```
git grep -w 16|grep qa/tasks/mgr/dashboard/test_pool.py
```

Reply:
done